lib/index.js

```diff
@@ -372,6 +394,28 @@ function opt (option) {
   }
 };
 
+/* chan and rmchan sockopt methods. only relevant for subscription sockets */
+Socket.prototype.chan       = function(){
```

Comment:
ok i was holding back since `nano.socket('sub', { chan: [''] })` is unacceptable... 
... as far as style goes.

but since you came up with that idea of `''` as the defualt where we run a `socket.chanrm('')` when passing in topic lists, let's definitely do that